docs/explanation/rocks.rst

```diff
@@ -3,22 +3,49 @@
 ROCKs
 =====
 
-ROCKs are container images that are compliant with the `Open Container
-Initiative`_ (OCI) `image format specification <OCI_image_spec_>`_.
-This means that ROCKs are interoperable with other tools that work with
-OCI-compliant images.
+ROCKs are **Ubuntu LTS-based container images** that are designed to meet
+cloud-native software's security, stability, and reliability requirements.
 
-A ROCK can be stored in an existing container registry, very much like any
-Docker image, and run in the same way as any other container image.
-For example, Docker can be used to run a ROCK with the usual command line
-syntax:
+ROCKs comply with the `Open Container Initiative`_'s (OCI) `image format
+specification <OCI_image_spec_>`_, and can be stored in any OCI-compliant
+container registry (e.g. DockerHub, ECR, ACR,..) and used by any OCI-compliant
+tool (e.g. Docker, Podman, Kubernetes,...).
 
-.. code:: bash
+Interoperability between ROCKs and other containers also extends to how
+container images are built. This enables using ROCKs as bases for existing
+build recipes, such as Dockerfiles, for further customisation and development.
 
-   docker run <rock> ...
+What sets ROCKs apart?
+~~~~~~~~~~~~~~~~~~~~~~
 
-Interoperability between ROCKs and other containers also extends to the way
-that container images are built. This enables the use of ROCKs as bases for
-existing build recipes, such as Dockerfiles, for further customisation and
-development.
+* **Opinionated and consistent design**: all ROCKs follow the same design,
+  aiming to minimise your full-stack disparity and adoption overhead, i.e
 
+  * :ref:`pebble_explanation_page` **is the official entrypoint for all
+    ROCKs**, thus providing a predictable and powerful abstraction layer
+    between the user and the container application;
+  * ROCKs extend the OCI image information by including additional **metadata**
+    inside each ROCK (e.g. at ``/.rock/metadata.yaml``), allowing container
+    applications to easily inspect the properties of the image they are running
+    on, at execution time;
+* **User-centric experience**: ROCKs are described in a :ref:`declarative
+  format<rockcraft.yaml_reference>` and **built on top of familiar and reliable
+  Ubuntu LTS images**, thus offering an open and up-to-date user experience;
+* **Seamless chiselling experience**: ROCKs can be effortlessly
+  :ref:`chiselled<chisel_explanation>` using off-the-shelf primitives,
+  harnessing all the advantageous traits of "distroless" to deliver **compact
+  and secure Ubuntu-based container images**.
+
+
+Design
```

Comment:
I think this section maybe going too fast, for a beginner who know nothing about OCI image you give too much information in 4 lines.

Reply:
Also i think adding somewhere the build diagram you have done is a good idea

Reply:
I've adjusted the section. Please have another look.

As for the diagram, we can discuss that in a future PR, but I feel it belongs in the Rockcraft section.